### ISSUE TITLE
Handle special keys

### ISSUE THREAD

**Issue by Manishearth** [chuckwagoncomputing/fxos-firemote#10]
Whilst sshing, the following keys are rather useful:
- Ctrl, Alt
- PgUp/PgDn (very useful for irssi), Home, End
- ESC (for vim)
- Arrow keys
- ins, del

What might be easiest is to provide Ctrl, Alt, keys at the topbar, with another key (or perhaps long-press?) which brings up a draggable keypad with nine keys -- arrow, pgup/pgdn/home/end, and ESC at the center. This is similar to what the SSH client on Windows Phone does and is quite usable.

**Comment by chuckwagoncomputing** [comment]
Personally, I think this should be implemented as a keyboard rather than a part of the app. @codingfree showed some interest in doing this. I would like to hear your opinions and the pros/cons of both ways.

**Comment by Manishearth** [comment]
All other SSH apps I know do this inline. Besides, a keyboard which you only use for one app sounds ... unweildy.

**Comment by chuckwagoncomputing** [comment]
Good point. I don't have time for this right now. If someone else wants to do it, go ahead.
